src/Runners/PHPUnit/WrapperRunner.php

```diff
@@ -125,4 +134,18 @@ private function waitForAllToFinish(): void
             usleep(self::CYCLE_SLEEP);
         }
     }
+
+    private function startWorker(int $token): void
+    {
+        $this->workers[$token] = new WrapperWorker($this->output, $this->options, $token);
+        $this->workers[$token]->start();
+        $this->batches[$token] = 0;
+    }
+
+    private function destroyWorker(int $token): void
+    {
+        $this->workers[$token]->stop();
+        $this->workers[$token]->__destruct();
+        unset($this->workers[$token]);
```

Comment:
```suggestion
        unset($this->workers[$token]);
```
`$this->workers` is the only reference to the object, so the `unset()` should be enough